src/Compositor.cpp

```diff
@@ -2431,7 +2431,7 @@ bool CCompositor::cursorOnReservedArea() {
     return !VECINRECT(CURSORPOS, XY1.x, XY1.y, XY2.x, XY2.y);
 }
 
-CWorkspace* CCompositor::createNewWorkspace(const int& id, const int& monid, const std::string& name) {
+CWorkspace* CCompositor::createNewWorkspace(const int& id, const int& monid, const std::string& name, bool isEmpty) {
```

Comment:
I don't like this approach. Very easy in the future to break the behavior.

Why not just set a flag in the compositor (needsRecheck-ish or something) and then verify this in sanityCheckWorkspaces?

if flag set, and there is an empty workspace that did not have the cmd ran yet (again, boolean for every workspace) then run it

Reply:
Addressed in 41a0734 and 1834dbc